lib/active_model/serializer.rb

```diff
@@ -37,6 +37,7 @@ class << self
       attr_accessor :_cache_except
       attr_accessor :_cache_options
       attr_accessor :_cache_digest
+      attr_accessor :_type
```

Comment:
I wish there were a better name for this, or a way to signal it is for json_api... `resource_type_name` ? no need to do anything